src/pages/Dashboard/widgets/BatchTransactions/types.ts

```diff
@@ -0,0 +1,6 @@
+export type SendTransactionProps = {
+  address: string;
+  nonce: number;
+  chainID: string;
```

Comment:
chainId

but this is no longer needed if you will make Transaction objects directly

Reply:
this is used for local components of passing props (not for transactions that are generated and being sent to be processed)